### ISSUE TITLE
es2015, es6?

### ISSUE THREAD

**Issue by amandapouget** [nadav-dav/rekuire#16]
We're trying to convert a project to using es2015 and Jest (based on `create-react-app`) and wondering if this package can have any place in it.... We used `rekuire` every in the old version and would have a lot of wood to move to take it out.

Thanks.

**Comment by nadav-dav** [comment]
Are you referring to the "import/export" syntax?

**Comment by amandapouget** [comment]
No. I’m saying, we have a legacy app which uses require everywhere whose files we’re trying to move on to a fresh foundation (made by `create-react-app`) which uses web pack and allows but does not require es2015 syntax. We’re trying to avoid having to spend days going through and changing all the `rekuire` statements to `import` statements.

We finally got `rekuire` ‘working’  for at least one file by making the following changes to `rekuire`’s main.js, though it’s essentially a mystery to us why this works:

```var baseDir = findBase(module.parent.paths);```
becomes
```var baseDir = path.resolve(findBase(module.parent.paths));```

and

```var parentReq = module.parent.require.bind(module);```
becomes
```
var parentReq;
if (module.parent.require) {
    parentReq = module.parent.require.bind(module);
} else {
    parentReq = require.bind(module);
}
```

Then the Jest tests run without a problem. Not sure exactly why the changes above ‘fix’ it… You can recreate the problem yourself by installing the `create-react-app` example app and trying to use `rekuire` in it.

**Comment by nadav-dav** [comment]
I'm glad you got it to run :) 
I am not really sure how the react app is related to the problem, but since you solved the issue, can you make a pull request so i'll merge it into the main repo?

thank you so much :)

**Comment by amandapouget** [comment]
Done.
